.github/workflows/register_repositories.py

```diff
@@ -0,0 +1,216 @@
+import asyncio
+import copy
+import json
+import logging
+import os
+from datetime import datetime
+
+import aiohttp
+import ruamel.yaml as yaml
+
+logging.basicConfig(level=logging.INFO)
+
+API_BASE_URL = 'https://api.github.com'
+
+XDEVBOT_MAIN_ENDPOINT = 'http://xdevbot.herokuapp.com/gh/testing'
+
+
+def parse_line(line, original_config, repos={'remove': [], 'add': []}):
+    config = copy.deepcopy(original_config)
+    valid_line = line.startswith('/add-repo') or line.startswith('/remove-repo')
+    if valid_line:
+        if line.startswith('/add-repo'):
+            split_on = '/add-repo'
+        else:
+            split_on = '/remove-repo'
```

Comment:
Now, I'm wondering if we want to modify this algorithm to allow for any kind of line *start*, but require that the command end with a line separator (i.e., `\n`).  That would allow people to actually list repos to add or remove in a list like:

```
- /add-repo campaign:campaign repo:repo1
- /add-repo campaign:campaign repo:repo2
- /remove-repo campaign:campaign repo:repo0
```

To do that, I think we might need:

```suggestion
    add_cmd = '/add-repo'
    remove_cmd = '/remove-repo'
    valid_line = add_cmd in line or remove_cmd in line
    if valid_line:
        split_on = add_cmd if add_cmd in line else remove_cmd
```

Would that work?

Reply:
...Maybe we would need `add_cmd = '/add-repo '` and `remove_cmd = '/remove-repo '`?